template/cordova/lib/log.js

```diff
@@ -151,23 +154,23 @@ function exitGracefully(exitCode) {
 }
 
 function startLogging(channel) {
-    var startTime = new Date().toISOString();
+    var logFromTime = 10 * ONE_MINUTE;
```

Comment:
This is set to the default value twice, [here][case1] and [there][case2], how come?

[case1]: https://github.com/apache/cordova-windows/pull/151/files#diff-378bd5207000848de6b771d61b037e2fR157
[case2]: https://github.com/apache/cordova-windows/pull/151/files#diff-378bd5207000848de6b771d61b037e2fR53